server/views/reponseToTriggerEmail.ejs

```diff
@@ -0,0 +1,22 @@
+<style>
+  button{
+     background:none;
+     border:none; 
+     padding:0;
+     font: inherit;
+     border-bottom:1px solid #000000; 
+     cursor: pointer;
+    }
+  </style>
+
+<h1><%= title %></h1>
+<% if (typeof content !== 'undefined') { %>
+  <p><%= content %></p>
+<% } %>
+
+<form action="<%= redirectToEmail %>" method='post'>
+   <button type='submit' value="<%= redirectToLinkText %>"><%= redirectToLinkText %></button>
+   for new email to complete registration or click 
+</form>
+
+<a href="<%= redirectTo %>">home</a> to login.
```

Comment:
1) Missing EOL at EOF (trailing newline), please fix.

2) I find it a bit unusual for a sentence to be broken by `</form>`. Is there any particular reason why this link cannot be included inside the `<form>` element?

```html
<form action="<%= redirectToEmail %>" method='post'>
  <button type='submit' value="<%= redirectToLinkText %>"><%= redirectToLinkText %></button>
  for new email to complete registration or click <a href="<%= redirectTo %>">home</a> to login.
</form>
```